custom_components/isy994/switch.py

```diff
@@ -77,6 +84,11 @@ def is_on(self) -> bool:
         """Get whether the ISY994 switch program is on."""
         return bool(self.value)
 
+    @property
+    def icon(self) -> str:
```

Comment:
If we're adding these two, we should add a different icon for variables (if not set already). Maybe `mdi:counter` or `mdi:numeric`?

Reply:
Looks like variables already defined one
```
    @property
    def icon(self):
        """Return the icon."""
        return self._config.get(CONF_ICON)
```